index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>Portifolio</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="styles.css">
+  </head>
+  <body>
+    <header class="section-navbar">
+      <nav>
+        <a href="#" class="toolbar-txt">Techmoves</a>
+
+          <a href="#" class="toggle-button">
+            <span class="bar"></span>
+            <span class="bar"></span>
+            <span class="bar"></span>
+          </a>
+
+      </nav>
+    </header>
+    <section id="headline">
+
+      <div class="back-img"></div>
+      
+      <h1>Hey There.<br>I'm Phelelani</h1>
+
+      <h2>I'm a Software Developer</h2>
+
+      <p id="introduction-txt">
+        I can help you build a product, feature or website Look through some of my work and experience! 
+        If you like what you see and have a project you need coded,
+        don't hestiate to contact me.
+      </p>
+      
+      <div class="social-media">
+        <ul>
+          <li class="link-git"><a href="https://github.com/techmoves"><img src="images\github.png" alt="GitHub icon"></a></li>
+           <li class="link-in"><a href="https://www.linkedin.com/in/ivan-ludick-11a055243/"><img src="images\linkedin-icon.png" alt="LinkedIn icon"></a></li>
+          <li class="link-twitter"><a href=https://twitter.com/Dopezonke_><img src="images\twitter.png" alt="Twitter icon"></a></li>
+          <li class="link-gmail"><a href="https://muhndlovu28@gmail.com"><img src="images\gmail.png" alt="Gmail icon"></a></li>
+        </ul>
+        <div class="backround-img"></div>
```

Comment:
- [OPTIONAL] Well done implementing the changes required by the previous reviewer. But I think that you are instead of taking a  `div` here and placing the background-image, placing the background-image to the hero section would be a better option.